src/models.ts

```diff
@@ -55,13 +41,11 @@ export type MutationHook = (
 export type ObjectModel = BaseModel & {
   type: 'object';
   interfaces?: string[];
-  // createdAt, createdBy, updatedAt, updatedBy can be null
-  nonStrict?: boolean;
```

Comment:
can be achieved using optional creatable/updatable/deletable field overrides below